### PR TITLE
PIEISSUE-6: fix for the wallet bug on mobile

### PR DESCRIPTION
**Pull request by TotoFullStackDev**
##### Description
you should now be able to connect from mobile, using any wallet of your choise.

##### Checklist

- [ ] Linter status: 100% pass
- [ ] Changes don't break existing behavior
- [ ] Test coverage hasn't decreased

### CODE REVIEW

File: src/stores/eth.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const providerOptions = {
   walletconnect: {
     package: WalletConnectProvider, // required
     options: {
-      infuraId: 'e106b2b27c0f4941be1f2c183a20b3ea', // required
+      infuraId: '1ec103a49691457aa6dff30aa8ab73d0', // e106b2b27c0f4941be1f2c183a20b3ea
```
Comment (Alexintosh) [inline src/stores/eth.js]: this has to change

**Review by Alexintosh — CHANGES_REQUESTED** [review]

**Comment by Alexintosh** [comment]
Seems to work well, there is a change requested and can you make the linter pass?

File: src/stores/eth.js (hunk 1 above)
Comment (TotoFullStackDev) [reply src/stores/eth.js]: yeah i know, just left it there to let you test.
i'm changing it :)

**Review by AdrianAddesso — APPROVED** [review]
Approved. 
Opened this separate issue to test on Mainnet after this PR and PR 279 are merged:
https://piedao.atlassian.net/browse/PIEISSUE-9

**Review by Alexintosh — APPROVED** [review]

**Pull request merged**